docs/pages/ktable.vue

```diff
@@ -86,12 +94,13 @@
         block
       />
 
-      <!--Table with default sort-->
       <h3>Table with default sort</h3>
```

Comment:
Also, should we add the section for `disableBuiltInSorting` before `defaultSort`, as we are already mentioning about the `disableBuiltInSorting` attribute here?

Reply:
Thank you, yes, I will swap 'Table with default sort' with 'Disable built-in sorting' (even though the default sort will hopefully soon work no matter of `disableBuiltinSorting` :)!